common-o2.sh

```diff
@@ -45,3 +48,7 @@ prepend-path LD_LIBRARY_PATH \$::env(COMMON_O2_ROOT)/lib
 $([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(COMMON_O2_ROOT)/lib")
 EoF
 mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+
+if [[ $ALIBUILD_O2_TESTS ]]; then
+  CTEST_OUTPUT_ON_FAILURE=1 ctest ${JOBS+-j $JOBS}
+fi
```

Comment:
Out of curiosity, why is it not enough to have this test in the incremental recipe?

Reply:
I wish I knew, I followed exactly how it's implemented in `o2.sh`

Reply:
Well, in general we want to run the tests also during a regular build, e.g. when building a release. Or am I misunderstanding your question? This of course is merely policy...@brinick can you check this actually builds fine and merge it?